client/packages.go

```diff
@@ -67,8 +67,31 @@ type Snap struct {
 }
 
 type AppInfo struct {
-	Name   string `json:"name"`
-	Daemon string `json:"daemon"`
+	Name         string `json:"name"`
+	DesktopFile  string `json:"desktop-file,omitempty"`
+	*ServiceInfo `json:",omitempty"`
+}
+
+// IsService returns true if the application is a background daemon.
+func (a *AppInfo) IsService() bool {
+	if a == nil {
+		return false
+	}
+	if a.ServiceInfo == nil {
+		return false
+	}
+	if a.ServiceInfo.Daemon == "" {
+		return false
+	}
+
+	return true
+}
+
+type ServiceInfo struct {
+	Daemon          string `json:"daemon"`
+	ServiceFileName string `json:"service-file-name"`
+	Enabled         bool   `json:"enabled"`
+	Active          bool   `json:"active"`
```

Comment:
this is a bit unfortunate because for snap Active is more similar to Enabled :/, not sure where we surface that though, except internally in snapstate